esmvalcore/preprocessor/_area.py

```diff
@@ -349,6 +349,27 @@ def _select_representative_point(shape, lon, lat):
     return select
 
 
+def _correct_coords_from_shapefile(cube, cmor_coords, natural_earth_file):
+    """Get correct lat and lon from shapefile."""
+    lon = cube.coord(axis='X').points
+    lat = cube.coord(axis='Y').points
+    if cube.coord(axis='X').ndim < 2:
+        lon, lat = np.meshgrid(lon, lat, copy=False)
+
+    if not cmor_coords:
+        # Wrap around longitude coordinate to match data
+        lon = lon.copy()  # ValueError: assignment destination is read-only
+        lon[lon >= 180.] -= 360.
+        if natural_earth_file:
```

Comment:
I don't see why you need to apply this to ne files. The ne files I got from https://www.naturalearthdata.com/downloads/ just run between the normal extends [-180, 180, -90, 90] so there is no need to wrap around the coordinates. And if some of them do wrap around, then the normal function would just work. 
I don't think there is anything different in natural earth shape files from other shape files, what I can see from them. I would propose to take this piece out.

Reply:
no, this is actually needed since w/o it we end up with misalignment: our current ne masks need that, also @LisaBock had issues with selecting and masking continents that crossed the prime meridian via ne mask files, also see https://github.com/ESMValGroup/ESMValTool/issues/590

Reply:
I don't get why you and @bouweandela are both so reluctant to this - it's checking if it needs wraparound and if not it's not doing it, even your docstring says `Note that this function does not work for shapes crossing the
    prime meridian or poles` so that will have to be removed once this gets in since it will work for regions crossing the prime meridian :grin:

Reply:
I am not reluctant about the wraparound, that's AWESOME! However the exception for ne files stings a bit. I just don't see how they are different from other shapefiles. If there is an issue, it is probably to also catch it in other possible cases, but for that we need to understand what the issue is.

Reply:
yeah absolutely! I am not familiar with shapefiles and the only ones Ive seen so far are these NE masks, which all needed this correction. The `ne_` string test is clearly dodgy but at the moment I don't know any better :grin:

Reply:
then it's maybe better if we check for the availability of coordinates at these points in the shapefile, Instead of using the dodgy `ne_` check.